.github/CHANGELOG.md

```diff
@@ -221,6 +221,8 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 
 * Ising XX gate functionality added. [(#1194)](https://github.com/PennyLaneAI/pennylane/pull/1194)
 
+* Added Projector observable, which is available on all devices inheriting from the `QubitDevice` class. [(#1356)](https://github.com/PennyLaneAI/pennylane/pull/1356)
```

Comment:
Nice 💪

Reply:
```suggestion
* Added Projector observable, which is available on all devices inheriting from the `QubitDevice` class.
  [(#1356)](https://github.com/PennyLaneAI/pennylane/pull/1356)
```

@wongwsvincent if you would, it would be great to add a minimal code example here to highlight how it is used for anyone reading the changelog after the next release. Have a look at other entries to see how the examples are formatted :)